app/__init__.py

```diff
@@ -151,6 +152,10 @@ def override_url_for():
     application.config['THEME_PATHS'] = os.path.dirname(os.path.abspath(__file__))
     Themes(application, app_identifier="surveyrunner")
 
+    @application.teardown_appcontext
+    def shutdown_session(exception=None):
```

Comment:
https://docs.pylint.org/en/1.6.0/features.html 

unused-argument (W0613):
 	Unused argument %r Used when a function or method argument is not used.
unused-variable (W0612):
 	Unused variable %r Used when a variable is defined but not used.